src/main/java/com/bankingservice/banking/exception/SuccessCode.java

```diff
@@ -0,0 +1,38 @@
+package com.bankingservice.banking.exception;
+
+import lombok.ToString;
+
+@ToString
+public enum SuccessCode {
+    /**
+     * Success code
+     */
+    SUCCESS_CODE("BS_ACC_20202", "Success");
```

Comment:
Add HttpStatusCode for this enum also, and use this enum in creating meta of success meta dto.